start-miner.sh

```diff
@@ -1,8 +1,16 @@
 #!/bin/sh
 
-wget \
-    -O /opt/miner/releases/0.1.0/sys.config \
-    https://helium-assets.nebra.com/docker.config
+if [[ -v OVERRIDE_CONFIG_URL ]];
+then
+    echo "Config URL Override Set!"
+    wget \
+         -O /opt/miner/releases/0.1.0/sys.config \
+         $OVERRIDE_CONFIG_URL
+else
+    wget \
+         -O /opt/miner/releases/0.1.0/sys.config \
+         https://helium-assets.nebra.com/docker.config
```

Comment:
Why not `${OVERRIDE_CONFIG_URL:=https://helium-assets.nebra.com/docker.config}`?